dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/Connector.java

```diff
@@ -36,10 +38,19 @@ public enum DefaultProperties implements ConnectorProperty {
     public String getName();
 
     @Nonnull
-    public String getDefaultFileName();
+    default String getDefaultFileName() {
+        SimpleDateFormat format = new SimpleDateFormat("yyyyMMdd'T'HHmmss");
+        Timestamp timestamp = new Timestamp(System.currentTimeMillis());
+        return "dwh-migration-" + getName() + "-" + type() + "-" + format.format(timestamp)
```

Comment:
The `format` method accepts also the milliseconds, so the wrapping with `Timestamp` is not required.

Reply:
Ah thanks! Didn't know this before. Applied!